plugin/WorkspaceSerializer.py

```diff
@@ -44,12 +52,14 @@ def serialize(self, version, value, context):
         context.write_using_serializer(string_serializer, value.name)
         context.write_using_serializer(string_serializer, value.description)
         context.write_using_serializer(vault_workspace_serializer, value.workspace)
+        context.write_using_serializer(interaction_array_serializer, value.interactions)
 
     def deserialize(self, version, context):
         name = context.read_using_serializer(string_serializer)
         description = context.read_using_serializer(string_serializer)
         workspace = context.read_using_serializer(vault_workspace_serializer)
-        return Scene(workspace, name, description)
+        interactions = context.read_using_serializer(interaction_array_serializer)
+        return Scene(workspace, name, description, interactions)
```

Comment:
Have you tested this with pre-interactions scene decks? It looks like those are gonna fail and not be loadable anymore